cli/flags.rs

```diff
@@ -103,31 +103,90 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
     ).subcommand(
       SubCommand::with_name("version")
         .setting(AppSettings::DisableVersion)
-        .about("Print the version"),
+        .about("Print the version")
+        .long_about(
+          "
+Print current version of Deno.
+
+Includes versions of Deno, V8 JavaScript Engine, and the TypeScript
+compiler.
+
+  deno version
+",
+        ),
     ).subcommand(
       SubCommand::with_name("fetch")
         .setting(AppSettings::DisableVersion)
         .about("Fetch the dependencies")
-        .arg(Arg::with_name("file").takes_value(true).required(true)),
+        .long_about(
+          "
+Fetch remote dependencies recursively.
```

Comment:
`fetch` compiles downloaded deps as well, I guess it's worth noting

Reply:
Agreed. I'll update this once I get more feedback on this (and after the pressure on travis has been alleviated)